src/components/videos/index.js

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios';
+import { useReducer } from 'react';
+import { useEffect } from 'react/cjs/react.development';
```

Comment:
I think only 'react' should be there, and you can import both together

Reply:
It was auto imported. I will update that.